code/modules/events/crystal_event.dm

```diff
@@ -109,14 +109,28 @@ This section is for the event controller
 	for(var/i in 1 to 6)
 		spawn_anomaly(spawners)
 
-	addtimer(CALLBACK(src, .proc/more_portals, GLOB.crystal_invasion_waves[wave_name]), 15 MINUTES)
+	var/list/crystal_spawner_turfs = list()
+	for(var/turf in RANGE_TURFS(6, dest_crystal.loc))
+		if(!isopenturf(turf) || isspaceturf(turf))
+			continue
+		var/turf/floor = turf
+		crystal_spawner_turfs += floor
```

Comment:
```suggestion
	for(var/range_turf in RANGE_TURFS(6, dest_crystal.loc))
		if(!isopenturf(range_turf) || isspaceturf(range_turf))
			continue
		crystal_spawner_turfs += range_turf
```
* No need to typecast here.
* Avoid variable names that can be confused with base type paths whenever possible.